PLUGINS.md

```diff
@@ -26,6 +26,5 @@ Inspect external instances (their ID and parsed XML) after parsing or provide cu
 
 ### API
 - `ExternalInstanceParser#addFileInstanceParser`
-- `ExternalInstanceparsser#addProcessor`
```

Comment:
It feels like `addFileInstanceParser` replaced this, so I wanted to take the opportunity to clean up the API.